docs/ros/config/yaml.mdx

```diff
@@ -6,3 +6,892 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
```

Comment:
Please run the prettier formatting on this file (from the readme -> you can format a single file by running npx prettier --write <FILE PATH> )

Reply:
Done, ran `prettier`